### PR TITLE
[ENGG-2280] feat: bluecore inspector

### PR DESCRIPTION
**Pull request by rohanmathur91**

### CODE REVIEW

File: browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEventPanel/VendorEventPanel.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Column, ResourceTable } from "@requestly-ui/resource-table";
 import { ObjectInspector } from "@devtools-ds/object-inspector";
 import { CollapseProps } from "antd/lib";
 import { capitalize } from "lodash";
+import { VendorEvent } from "@requestly/analytics-vendors/types";
```
Comment (nafees87n) [inline browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEventPanel/VendorEventPanel.tsx:9]: We should export the types from `@requestly/analytics-vendors` index.js
as this wont work when npm package is published.

```js
// analytics-vendors index.ts
export * from types;
```

and then here 

```suggestion
import { VendorEvent } from "@requestly/analytics-vendors";
```

For reference: check the structure of requestly-web-sdk's root file

**Review by nafees87n — CHANGES_REQUESTED** [review]

File: browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEventPanel/VendorEventPanel.tsx (hunk 1 above)
Comment (rohanmathur91) [reply browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEventPanel/VendorEventPanel.tsx:9]: Thanks for pointing out!

**Review by nafees87n — APPROVED** [review]
Looks good

File: common/analytics-vendors/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "analytics-vendors",
+  "version": "1.0.0",
+  "description": "\"List of popular analytics vendors, their Identifiers and helps you test, debug and validate the analytics implementation\"",
```
[the remark this answers is not in the document]
Comment (rohanmathur91) [reply common/analytics-vendors/package.json:4]: Yes

**Pull request merged**